docs/source/contents/inference/index.md

```diff
@@ -1,107 +1,375 @@
 # Inference
 
-This section will discuss how to make inference calls against your seldon models or pipelines.
+This section will discuss how to make inference calls against your Seldon models or pipelines.
 
 You can make synchronous inference requests via REST or gRPC or asynchronous requests via Kafka topics.
+The content of your request should be an [inference v2 protocol payload](../apis/inference/v2.md):
+* REST payloads will generally be in the JSON v2 protocol format.
+* gRPC and Kafka payloads **must** be in the Protobuf v2 protocol format.
```

Comment:
do we provide examples how we can write directly to a kafka topic?

Reply:
I don't think we have any such examples anywhere currently, at least not proper ones.
It's probably one to add an issue for/add to the roadmap.

There might be some under `scheduler/data-flow/scripts`, but they'd need to be checked as still being valid and they're not in Python, so less notebook friendly.

Reply:
A follow up ticket if not easily available makes sense.